gnomad/utils/vep.py

```diff
@@ -423,6 +444,45 @@ def filter_vep_to_synonymous_variants(
     )
 
 
+def filter_vep_to_gene_list(
+    t: Union[hl.MatrixTable, hl.Table],
+    genes: List[str],
+    match_by_gene_symbol: bool = False,
+    vep_root: str = "vep",
+    filter_empty_csq: bool = False,
+):
+    """
+    Filter VEP transcript consequences to those in a set of genes.
+
+    .. note::
+
+       Filtering to a list of genes by their gene_id or gene_symbol will filter to
+       all variants that are annotated to the gene, including
+       ['upstream_gene_variant', 'downstream_gene_variant'], which will not be the
+       same as if you filter to a gene interval. If you only want variants inside
+       certain gene boundaries and filter faster, you can filter the variant first to an
+       interval list and further filter to specific genes.
```

Comment:
```suggestion
       certain gene boundaries and a faster filter, you can first filter `t` to an interval list and then apply this filter.
```

certain gene boundaries and a faster filter, you can first filter `t` to an